helm/config-controller/templates/rbac.yaml

```diff
@@ -5,52 +5,16 @@ metadata:
   labels:
     {{- include "labels.common" . | nindent 4 }}
 rules:
-  - apiGroups:
-      - ""
-    resources:
-      - pods
-    verbs:
-      - "*"
-  - apiGroups:
-      - ""
-    resources:
-      - configmaps
-      - namespaces
-    verbs:
-      - create
-      - update
-      - delete
-      - get
-      - list
-  - apiGroups:
-      - ""
-    resources:
-      - serviceaccounts
-    verbs:
-      - create
-  - apiGroups:
-      - "networking.k8s.io"
-    resources:
-      - networkpolicies
-    verbs:
-      - create
-  - apiGroups:
-      - "rbac.authorization.k8s.io"
-    resources:
-      - clusterrolebindings
-    verbs:
-      - create
-  - apiGroups:
-      - apiextensions.k8s.io
-    resources:
-      - customresourcedefinitions
-    verbs:
-      - "*"
-  - nonResourceURLs:
-      - "/"
-      - "/healthz"
-    verbs:
-      - get
+- apiGroups:
+    - application.giantswarm.io
+  resources:
+    - apps
+  verbs:
+    - get
+    - list
+    - patch
```

Comment:
For my curiosity, why do we need perms to patch / update App CRs? what changes does the config-controller make to App CRs?

Reply:
Sets CM and Secret